tools/pythonpkg/tests/fast/test_all_types.py

```diff
@@ -378,5 +378,6 @@ def test_pandas(self, duckdb_cursor):
             print(cur_type)
```

Comment:
why you have the prints? they are not really part of the test right

Reply:
I think they were already there, no?
It helps with debugging anyways, when these tests fail they are not super easy to figure out what is actually happening.